mediapipe/tasks/ios/vision/core/sources/MPPTaskImage.h

```diff
@@ -0,0 +1,178 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <CoreMedia/CoreMedia.h>
+#import <CoreVideo/CoreVideo.h>
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** Types of image sources. */
+typedef NSInteger MPPTaskImageSourceType NS_TYPED_ENUM NS_SWIFT_NAME(TaskImageSourceType);
+/** Image source is a `UIImage`. */
+static const MPPTaskImageSourceType MPPTaskImageSourceTypeImage = 0;
+/** Image source is a `CVPixelBuffer`. */
+static const MPPTaskImageSourceType MPPTaskImageSourceTypePixelBuffer = 1;
+/** Image source is a `CMSampleBuffer`. */
+static const MPPTaskImageSourceType MPPTaskImageSourceTypeSampleBuffer = 2;
+
+/** An image used in on-device machine learning using MediaPipe Task library. */
+NS_SWIFT_NAME(MPPTaskImage)
+@interface MPPTaskImage : NSObject
```

Comment:
Please name it MPPImage for Obj-C and MPImage for Swift (to avoid collision with SwiftUI's [Image](https://developer.apple.com/documentation/swiftui/image))

Reply:
Done